jinahub/indexers/query/vector/AnnoyIndexer/requirements.txt

```diff
@@ -1 +1,4 @@
 annoy==1.17.0
+jina==2.0.0rc5.dev54
```

Comment:
It is curious in `docker` we ask for `master`

Reply:
we don't yet have a proper Docker base image for the tests. We just whatever image and then install the reqs in one of the steps